src/openrct2-ui/WindowManager.cpp

```diff
@@ -1380,7 +1383,9 @@ class WindowManager final : public IWindowManager
 
     WindowBase* BringToFrontByClass(WindowClass cls) override
     {
-        return BringToFrontByClassWithFlags(cls, WF_WHITE_BORDER_MASK);
+        WindowFlags flags{};
+        WindowFlagsSetFlashCountDown(flags);
```

Comment:
This function `WindowFlagsSetFlashCountDown` seems to not be a direct translation of the `WF_WHITE_BORDER_MASK` flag. Instead, both `WindowFlag::whiteBorderOne`, `WindowFlag::whiteBorderTwo` are set. Is this intentional?

Reply:
It _is_ a direct translation. Note that this flag was defined as follows:
```C++
WF_WHITE_BORDER_MASK = (1 << 12) | (1 << 13),
```